extensions/amp-viewer-integration/0.1/test/test-highlight-handler.js

```diff
@@ -409,57 +412,79 @@ describes.realWin.skip(
       expect(setScrollTopStub).to.be.calledOnce;
       expect(setScrollTopStub.firstCall.args[0]).to.equal(350);
     });
+  }
+);
 
+describes.realWin(
+  'HighlightHandler',
```

Comment:
Should not have the same name as the test suite above

Reply:
Actually this is intentional. It gets formatted pretty nicely on the console:

![Screen Shot 2021-09-14 at 10 37 28 AM](https://user-images.githubusercontent.com/506183/133307458-5fdefd67-1369-4ddd-8cfc-3164d170aa8a.png)

Reply:
Ah, I had no idea. Wonderful